ArtOfIllusion/src/artofillusion/PluginRegistry.java

```diff
@@ -203,6 +204,12 @@ private static void processJar(JarInfo jar, Map<String, JarInfo> nameMap)
         registerResource(info.type, info.id, jar.loader, info.name, info.locale);
       }
     }
+    catch(NoClassDefFoundError ncdfe)
```

Comment:
As previously mentioned, I think we do need to broaden this, at least to `LinkageError` if not all the way to just `Error`.